misk/src/main/kotlin/misk/metrics/MetricsModule.kt

```diff
@@ -0,0 +1,12 @@
+package misk.metrics
+
+import com.codahale.metrics.MetricRegistry
+import com.google.inject.AbstractModule
+import com.google.inject.Singleton
+
+class MetricsModule : AbstractModule() {
+  override fun configure() {
+    bind(MetricRegistry::class.java).toInstance(MetricRegistry())
+    bind(Metrics::class.java).`in`(Singleton::class.java)
```

Comment:
We have an extension function in `Guice.kt` called `asSingleton()` you can use here.